### DISCUSSION TITLE
Component causing elements with wire:transition in other component to disappear

### DISCUSSION THREAD

**Discussion by jolora** [livewire/livewire#7672] [1. Help]
I'm looking for some help please with a strange problem.

In my application I added a second component to a view only to find an element within the first component would disappear whenever a property in the second component was changed.

I've stripped the problem back to its basics in a brand new Laravel app and the unexpected behaviour is still present. Here's my code:

**View:**
```
<!DOCTYPE html>
<html>
    <head>
	@livewireStyles
    </head>
    <body>
	<livewire:test />
	<livewire:test2 />
	@livewireScripts
    </body>
</html>
```

**Component 1:** 
```
<?php

namespace App\Livewire;

use Livewire\Component;

class Test extends Component
{
  public function render()
    {
      return view('livewire.test');
    }
}
```

**Component 1 view:**

```
<div>
	@if(true)
	<strong>First component</strong>
		<div wire:transition>
			Element <em>with</em> `wire:transition` attribute.
		</div>
		<div>
			Element <em>without</em> `wire:transition` attribute.
		</div>
	@endif
</div>
```

**Component 2:**
```
<?php

namespace App\Livewire;

use Livewire\Component;

class Test2 extends Component
{
	public $value = true;

	public function changeValue()
	{
		$this->value = !$this->value;
	}

    public function render()
    {
        return view('livewire.test2');
    }
}
```

**Component 2 view:**

```
<div wire:click="changeValue">
	@if($value == true)
		<div>
			<button>Second component.</button>
		</div>
	@endif
</div>
```

Whenever the 'changeValue' method is run the element in component 1 with the wire:transition attribute disappears.

I feel like I'm missing something obvious but at the moment I'm just going in circles.

I've created a quick repo with a vanilla Laravel install to show the problem here: https://github.com/jolora/livewire-bug

Any advice would be greatly appreciated please.

**Comment by jolora** [comment]
Mistake was a stupid one... and the code above was totally irrelevant 🤦‍♂️

**Reply by volkeransmann** [reply]
Hi @jolora : I am seeing the same behaviour. Could you tell me about your mistake?

**Reply by jolora** [reply]
> Hi @jolora : I am seeing the same behaviour. Could you tell me about your mistake?

If memory serves me correctly I hadn't actually resolved the original problem (although briefly thought I had). I instead switched to using Alpine directly to handle the transitions that were failing. I never resolved the underlying problem with my Livewire code though.

**Reply by volkeransmann** [reply]
Thanks for your quick reply. I think i will go the same route. Not enough time to debug right now. But very, very strange.
